packages/execution-environments/src/common/endowments/timeout.ts

```diff
@@ -18,25 +18,27 @@ const createTimeout = () => {
       );
     }
 
-    const handle = setTimeout(() => {
-      registeredTimeouts.delete(handle);
+    const handle = Object.freeze({});
```

Comment:
```suggestion
    const opaqueHandle = Object.freeze({});
```

Reply:
Would like to run with `handle` rather than `opaqueHandle`. Seems less confusing.